draft/spec/index.html

```diff
@@ -1031,6 +1031,43 @@ <h2>Versioned OCFL Object</h2>
 </pre>
     </section>
 
+    <section id="example-object-with-file-renaming">
+        <h2>File Renaming in an OCFL Object</h2>
```

Comment:
Is this really file renaming?  Or is this more about file names

Reply:
Yes, I think based on feedback the term renaming title is wrong. I'll try (using the current/old terminology) "Different Existing File Paths and Logical File Paths in an OCFL Object" or (using the new terminology) "Different Storage Paths and Logical Paths in an OCFL Object".